recipe/meta.yaml

```diff
@@ -19,13 +19,15 @@ requirements:
     - cmake
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - ninja
+    - ninja-base
   host:
-    - aws-c-common 0.6.8
-    - aws-checksums 0.1.11
+    - aws-c-common 0.8.5
```

Comment:
I'm wondering why these packages are so tightly coupled to each other at the minor version. Do they all get tied together this tightly upstream? and is there really no api compatibility across patch versions?

Reply:
they are vendored and referenced as submodules with exact tags, we want to unvendor it

Reply:
the submodules need to be tagged to a certain commit though, I don't think this implies no API compatibility across versions?
Are these packages used for anything else except pyarrow, i.e. would loosening this help compatibility/installability?

Reply:
We could easily pin on major+minor. But it's definitely safer to pin than not to pin when it comes to AWS SDKs (or any cloud vendor SDKs).

Reply:
Though, they all have strict `run_exports`, which implies instabilities.

Reply:
yeah, I was wondering where those came from.